streamlit_folium/__init__.py

```diff
@@ -271,6 +271,8 @@ def st_folium(
 
     folium_map: folium.Map = fig  # type: ignore
 
+    folium_map.render()
```

Comment:
Is this a change to folium? If so, do we need to change the setup.py versions?

Reply:
Yes, this seems to be related to a change in Folium, in particular how it handles adding map layers to the map. I think everything I did should be backwards-compatible, so it should be fine to leave open-ended requirements in setup.py. Ideally we would have a matrix that would test on a few different versions of folium & streamlit, but that would get real slow real fast.

Reply:
> I think everything I did should be backwards-compatible

Doesn't seem like that's possible, if the previous versions don't work. Or are you saying this is the way it could've/should've always worked, we just assumed too much?

Reply:
Also, I noticed about 500ms of new latency...